src/Molecules/Header/Header.test.jsx

```diff
@@ -0,0 +1,189 @@
+import React, { useState, cloneElement, Children } from 'react'
+import { MemoryRouter as Router } from 'react-router'
+import { ThemeProvider } from 'styled-components'
+import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles'
+import { render, act, fireEvent } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import user from '@testing-library/user-event'
+
+import { muiRg6Theme } from '../../../.storybook/themes'
+import Header from './index'
+
+// FIXME: MuiThemeProvider and ThemeProvider are required here because of Tooltip theme usage
```

Comment:
🤔

Reply:
This looks broken to me, that's why I've added this `FIXME`